scene/gui/control.cpp

```diff
@@ -1538,7 +1538,7 @@ Control::LayoutMode Control::_get_layout_mode() const {
 	}
 	// Otherwise check what was saved.
 	if (has_meta("_edit_layout_mode")) {
-		return (LayoutMode)(int)get_meta("_edit_layout_mode");
+		return (LayoutMode)int(get_meta("_edit_layout_mode", DEFAULT_LAYOUT_MODE));
 	}
 	// Or fallback on default.
 	return LayoutMode::LAYOUT_MODE_POSITION;
```

Comment:
The `if has_meta()` part is not necessary, `DEFAULT_LAYOUT_MODE` will be returned if it doesn't exist, so you can change all this to a single line.

Reply:
Well noted.